src/components/Prompt.spec.js

```diff
@@ -4,7 +4,7 @@ import App from './App'
 
 it('renders the Prompt component', () => {
   render(<App />)
```

Comment:
This is the `Prompt` test, I think it would be better if you render `<Prompt />` and check if it is visible. But as it is just a styled component, you could write these tests in your `App.spec.js` and delete this test :)

Reply:
Thank you, will do! Testing is the bain of my existence. Cypress is a lot more intuitive and I hope it will make it a bit quicker for me. I write the functional code for my user stories in about 3-4h depending on the complexity but then Testing in Jest and Storybooking add 12-20h on top because I can't seem to make the jest tests work for me. 😭
But I think the prop drilling tip you gave further below for Storybook will help a lot already! ☺️

Reply:
I am sorry to hear that testing and storybook are such a source of frustration for you! Would it help if we'd do a Q&A session for testing and/or storybook to clarify some issues?